group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorService.java

```diff
@@ -523,7 +524,7 @@ public CompletableFuture<ListGroupsResponseData> listGroups(
             .combineFutures(futures, ArrayList::new, List::addAll)
             .thenApply(groups -> new ListGroupsResponseData().setGroups(groups))
             .exceptionally(exception -> handleOperationException(
-                "ListGroups",
+                "list-groups",
```

Comment:
this covers class and consumer groups?

Reply:
Indeed, list groups lists all the groups.